ext/com_dotnet/com_misc.c

```diff
@@ -72,7 +71,6 @@ PHP_COM_DOTNET_API void php_com_wrap_variant(zval *z, VARIANT *v,
 	obj = emalloc(sizeof(*obj));
 	memset(obj, 0, sizeof(*obj));
 	obj->code_page = codepage;
-	obj->ce = php_com_variant_class_entry;
 	obj->zo.ce = php_com_variant_class_entry;
```

Comment:
Also likely superfluous.

Reply:
Ditto, this is set by zend_object_std_init below